src/components/Button/Button.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components/macro'
```

Comment:
I always forget to add the `/macro` and wonder why my classes look weird... 😅 👍🏻

Reply:
 i'm still waiting to write 'marco' ... 😜

Reply:
i'm still 😜waiting to write 'marco' ...